content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -213,18 +191,63 @@ The `from-key` value is specific to the integration type, and should exclude the
 **Default**: `true`
 
 `DD_HTTP_SERVER_TAG_QUERY_STRING_SIZE`
-: When `DD_HTTP_SERVER_TAG_QUERY_STRING` is true, the max size of the querystring to report is set, before obfuscation. Set 0 for no limit in size<br>
+: When `DD_HTTP_SERVER_TAG_QUERY_STRING` is true, sets the max size of the querystring to report, before obfuscation. Set 0 for no limit in size<br>
 **Default**: `5000`
 
 `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP`
 : When `DD_HTTP_SERVER_TAG_QUERY_STRING` is true, this regex redacts sensitive data from incoming requests' query string reported in the `http.url` tag (matches are replaced with `<redacted>`). This regex executes for every incoming request.
 
+#### Agent
```

Comment:
Should we call this something like "Agent-related" to avoid confusion, since they are not settings for the Agent itself?